config/routes.rb

```diff
@@ -10,4 +10,6 @@
 
   # Uncomment this line to activate the endpoint for the Square order.updated Webhook
   # post "webhooks/square", to: '/solidus_square/webhooks#update'
+
+  post "payments/square", to: '/solidus_square/payments#create'
```

Comment:
this should be under API path

Reply:
https://github.com/nebulab/solidus_square/pull/49/commits/b0dd811d394c6ce3eaf59f85907c46318ebb244d